content/blog/2020-outreach-partners.md

```diff
@@ -0,0 +1,31 @@
+Title: Bringing PyCon India to a wider audience with your help
+Date: 2020-07-23 13:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-outreach-partners
+
+[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about the conference in their communities and help bring more fresh faces to the event.

```

Comment:
```suggestion
[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about our conference in communities all over the world and bring fresh faces to the platform in hopes of creating a more inclusive, accessible platform this year.
```

Reply:
Can we use this but break it down into 2 sentences. If that helps

Reply:
I think the single sentence looks better. Also, how about `creating a more inclusive, accessible conference this year`

Reply:
Works